app/src/main/res/values/strings.xml

```diff
@@ -596,6 +596,9 @@ like to connect to:</string>
   <string name="tile_missing_entity_title">Missing Script or Scene</string>
   <string name="tile_missing_entity_summary">Please add a script or scene to Home Assistant and revisit this screen to enable the below field</string>
   <string name="template_widget">Template Widget</string>
-    <string name="basic_sensor_name_high_accuracy_mode">High Accuracy Mode</string>
+  <string name="basic_sensor_name_high_accuracy_mode">High Accuracy Mode</string>
   <string name="sensor_description_high_accuracy_mode">Whether or not high accuracy mode is active on the device</string>
+  <string name="sensor_name_device_policy">Device Policy</string>
+  <string name="sensor_name_work_profile">Work Profile</string>
+  <string name="sensor_description_work_profile">Whether or not the work profile is currently active on the device</string>
```

Comment:
I know that you have just copied the format of existing description strings, but this form is arguably incorrect.  It should not include "or not", since that changes the meaning to "Regardless of whether the work profile is currently active on the device".  The correct description should simply be "Whether the work profile is currently active on the device".

One such discussion of the difference is at https://www.grammarphobia.com/blog/2013/03/whether-or-not.html